product_warranty_management/models/product_template.py

```diff
@@ -0,0 +1,9 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+     
+    is_warranty_available = fields.Boolean(string="Is Warranty Available", default=False)
```

Comment:
```suggestion
    is_warranty_available = fields.Boolean(string="Is Warranty Available")
```
Need of explicitly defining the default value? :thinking: